### ISSUE TITLE
(Buff)Heavy Plasma Torpedo M1

### ISSUE THREAD

**Issue by reekafozoM** [PavelZinchenko/event-horizon-frontier-builds#349]
I suggest adding mechanics to this weapon so that it can stick to some enemy object: Starbase or enemy ship. or increase the damage of this gun by 50 or even 100 percent. After all, without a large number of damage amplifiers, this weapon is very ineffective.

**Comment by SuplexCity25284** [comment]
It's all together too slow, and too randomly targeted to be effectively used.  By the time you fire, the enemy is gone.  The other similar type weapon homes in on enemies, but it very slow and doesn't have much range.  Both at anything about around level 100 are tantamount to useless

**Comment by reekafozoM** [comment]
I know it's not good for destroying ships. But it's perfect for destroying a starbase. - This should work in the SB arena.

**Comment by SuplexCity25284** [comment]
I agree with you there.  I wouldn't mind it it worked like a sticky grenade in FPS games.  It hits sticks, then after a while detonates.   It would be a good weapon against larger ships too, since they usually slower
